### PR TITLE
[abseil] Update to 20200923.2 version

### PR DESCRIPTION
**Pull request by cngzhnp**
Update **abseil** version to the 20200923.2 version.

**Important note:** A few patches are not necessary anymore because all of them are fixed with the latest abseil version.

### CODE REVIEW

**Review by JackBoosY — CHANGES_REQUESTED** [review]
Please bump the port version. See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).

File: ports/abseil/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "abseil",
-  "version-string": "2020-03-03",
-  "port-version": 8,
+  "version-string": "2020-10-21",
+  "port-version": 0,
```
Comment (JackBoosY) [inline ports/abseil/vcpkg.json:4]:
```suggestion
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Comment by JackBoosY** [comment]
```
     5>time_zone_info.obj : error LNK2001: unresolved external symbol "class std::unique_ptr<class absl::lts_2020_09_23::time_internal::cctz::ZoneInfoSource,struct std::default_delete<class absl::lts_2020_09_23::time_internal::cctz::ZoneInfoSource> > (__cdecl* absl::lts_2020_09_23::time_internal::cctz_extension::zone_info_source_factory)(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::function<class std::unique_ptr<class absl::lts_2020_09_23::time_internal::cctz::ZoneInfoSource,struct std::default_delete<class absl::lts_2020_09_23::time_internal::cctz::ZoneInfoSource> > __cdecl(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)> const &)" (?zone_info_source_factory@cctz_extension@time_internal@lts_2020_09_23@absl@@3P6A?AV?$unique_ptr@VZoneInfoSource@cctz@time_internal@lts_2020_09_23@absl@@U?$default_delete@VZoneInfoSource@cctz@time_internal@lts_2020_09_23@absl@@@std@@@std@@ABV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@6@ABV?$function@$$A6A?AV?$unique_ptr@VZoneInfoSource@cctz@time_internal@lts_2020_09_23@absl@@U?$default_delete@VZoneInfoSource@cctz@time_internal@lts_2020_09_23@absl@@@std@@@std@@ABV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@2@@Z@6@@ZA) [D:\buildtrees\abseil\arm-uwp-dbg\absl\abseil_dll.vcxproj]
```

**Comment by cngzhnp** [comment]
I will close this PR because of unexpected problems that I faced in my environment,

**Pull request closed without merging**